schema/derive/src/lib.rs

```diff
@@ -192,7 +192,7 @@ fn metadata_for_enums(data_enum: &DataEnum) -> (Vec<Type>, Expr) {
             );
             quote! {
                 iroha_schema::EnumVariant {
-                    name: stringify!(#name).to_owned(),
+                    name: String::from(stringify!(#name)),
```

Comment:
Not a comment, more of a question, is this change necessary for `no-std` support? AFAIK the performance impact is minimal, since the blanket `impl` isn't too bad, but still...

Reply:
yeah, it's not about the performance, it's about dependencies. With `to_owned` you would also have to include`use alloc::string::ToOwned` when using this macro. So the reason is to reduce the number of required imports. I think you have to import `String` and `Vec` thought